install/odh/base/deployment.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: odh-dashboard
+spec:
+  replicas: 2
```

Comment:
Should we just deploy 1 by default?

Reply:
We can, there's no failover redundancy, though.

Reply:
@vpavlin leaving as 2 for now since NodeJS being single threaded going down to 1 scares me. Of course, these templates are just for example when ported over to the operator, so we can discuss then.